src/renderer/dx/DxRenderer.cpp

```diff
@@ -1196,21 +1101,19 @@ void DxEngine::_InvalidOr(RECT rc) noexcept
 // - S_OK
 [[nodiscard]] HRESULT DxEngine::PaintBackground() noexcept
 {
-    switch (_chainMode)
-    {
-    case SwapChainMode::ForHwnd:
-        _d2dRenderTarget->FillRectangle(D2D1::RectF(static_cast<float>(_invalidRect.left),
-                                                    static_cast<float>(_invalidRect.top),
-                                                    static_cast<float>(_invalidRect.right),
-                                                    static_cast<float>(_invalidRect.bottom)),
-                                        _d2dBrushBackground.Get());
-        break;
-    case SwapChainMode::ForComposition:
-        D2D1_COLOR_F nothing = { 0 };
+    D2D1_COLOR_F nothing = { 0 };
 
+    // Runs are counts of cells.
+    // Use a transform by the size of one cell to convert cells-to-pixels
+    // as we clear.
+    _d2dRenderTarget->SetTransform(D2D1::Matrix3x2F::Scale(_glyphCell));
+    for (const auto rect : _invalidMap.runs())
+    {
+        _d2dRenderTarget->PushAxisAlignedClip(rect, D2D1_ANTIALIAS_MODE_ALIASED);

```

Comment:
you asked carlos to add comments to `ALIASED` sooooo

Reply:
yeah good call.